tests/test_hexary_trie.py

```diff
@@ -84,9 +88,17 @@ def test_trie_using_fixtures(fixture_name, fixture):
         in keys_and_values
         if k not in deletes
     }
+    return remaining, deletes
+
+
+@pytest.mark.parametrize(
+    'fixture_name,fixture', FIXTURES,
+)
+def test_trie_using_fixtures(fixture_name, fixture):
 
-    for kv_permutation in itertools.islice(itertools.permutations(keys_and_values), 100):
-        print("in it")
+    remaining, deletes = get_expected_results(fixture)
+
+    for kv_permutation in get_fixture_iterator(fixture):
```

Comment:
Thoughts on using a parameterized fixture instead of this loop so that we get a clearer definition of which permutation(s) fail?

Reply:
I'm not opposed. Any failure ends up printing kv_permutation, and this whole batch only takes ~1s to run, so it wasn't a major barrier to getting work done. But that a test for each permutation is more stylistically consistent. :balance_scale: